rhel8/profiles/ism_o.profile

```diff
@@ -0,0 +1,144 @@
+  - disable_host_auth
+  - require_emergency_target_auth
+  - require_singleuser_auth
+  - sebool_authlogin_nsswitch_use_ldap
+  - sebool_authlogin_radius
+  - sshd_disable_kerb_auth
+  - sshd_set_max_auth_tries
+  - sssd_enable_smartcards
+  - accounts_password_minlen_login_defs
+  - var_password_pam_minlen=14
+  - accounts_password_pam_minlen
+  - accounts_password_pam_minclass
+  - accounts_password_pam_dcredit
+  - accounts_password_pam_lcredit
+  - accounts_password_pam_ocredit
+  - accounts_password_pam_ucredit
+  - accounts_password_pam_maxrepeat
+  - accounts_passwords_pam_faillock_deny
+  - accounts_passwords_pam_faillock_deny_root
+  - accounts_passwords_pam_faillock_interval
+  - accounts_passwords_pam_faillock_unlock_time
+
+  ## Password authentication & Protecting credentials
+  ## Identifiers 1055 / 0418 / 1402
+  - network_nmcli_permissions
+  - configure_kerberos_crypto_policy
+  - kerberos_disable_no_keytab
+  - sebool_kerberos_enabled
+  - sshd_disable_gssapi_auth
+  - enable_ldap_client
+  - set_password_hashing_algorithm_libuserconf
+  - set_password_hashing_algorithm_logindefs
+  - set_password_hashing_algorithm_systemauth
+  - accounts_password_warn_age_login_defs
+  - accounts_maximum_age_login_defs
+  - accounts_minimum_age_login_defs
+ 
+  ## System administration & MFA
+  ## Identifiers 1382 / 1384 / 1386
+  - package_sudo_installed
+  - configure_opensc_card_drivers
+  - force_opensc_card_drivers
+  - sebool_authlogin_yubikey
+
+  ## System patching & Applicatoin versions
+  ## Identifiers 1493 / 1144 / 0940 / 1472 / 1494 / 1495 / 1467 / 1483 
+  - dnf-automatic_apply_updates
+  - package_dnf-plugin-subscription-manager_installed
+  - package_subscription-manager_installed
+
+  ## Centralised logging facility
+  ## Identifiers 1405 / 0988 
+  - rsyslog_cron_logging
+  - rsyslog_files_groupownership
+  - rsyslog_files_ownership
+  - rsyslog_files_permissions
+  - rsyslog_nolisten
+  - rsyslog_remote_loghost
+  - rsyslog_remote_tls
+  - rsyslog_remote_tls_cacert
+  - package_chrony_installed
+  - service_chronyd_enabled
+  - chronyd_or_ntpd_specify_multiple_servers
+  - chronyd_specify_remote_server
+  - service_chronyd_or_ntpd_enabled
+
+  ## Events to be logged
+  ## Identifiers 0584 / 0582 / 0585 / 0586 / 0846 / 0957
+  - display_login_attempts
+  - sebool_auditadm_exec_content
+  - audit_rules_privileged_commands
+  - audit_rules_session_events
+  - audit_rules_unsuccessful_file_modification
+  - audit_access_failed
+  - audit_access_success
+
+  ## Web application & Database servers
+  ## Identifiers 1552 / 1277
+  - openssl_use_strong_entropy
+
+  ## Network design and configuration
+  ## Identifiers 1311 
+  - service_snmpd_disabled
+  - snmpd_use_newer_protocol
+
+  ## Wireless networks
+  ## Identifiers 1315 / 1319
+  - wireless_disable_interfaces
+  - network_ipv6_static_address
+
+  ## ASD Approved Cryptopgraphic Algorithims
+  ## Identifiers 1446
+  - enable_dracut_fips_module
+  - enable_fips_mode
```

Comment:
For this rule to pass, you have to also include:
    - var_system_crypto_policy=fips_ospp
    - configure_crypto_policy
This will configure system-vide crypto policy to "fips". Make sure that this IS what you want.

Reply:
Updated via https://github.com/wcushen/content/commit/c05588a4b3f56c74fae1297fc1562a0608e61f35
I've opted for `var_system_crypto_policy=fips` not `fips_ospp`. Is that OK @vojtapolasek?

Reply:
Updated again to https://github.com/wcushen/content/commit/dbc698d93bbc3593bc5ff3cf526c94cbf02f42c8 to align with `default_nosha1` set Essential Eight RHEL 8 (https://github.com/ComplianceAsCode/content/blob/4d3d54454c95c762533ac34ced6d8f3255378a48/rhel8/profiles/e8.profile#L135). I thought as it extends this profile (E8) that that var would have been picked up?

Reply:
@vojtapolasek Now been updated to specify `fips`
https://github.com/wcushen/content/blob/master/rhel8/profiles/ism_o.profile#L145